src/Playwright/Transport/WebSocketTransport.cs

```diff
@@ -0,0 +1,199 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.WebSockets;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Playwright.Transport
+{
+    internal class WebSocketTransport : IConnectionTransport, IDisposable
+    {
+        private const int DefaultBufferSize = 1024;  // Byte buffer size
```

Comment:
Bump it to 16K?

Reply:
yup, that also works, I think no need for looping messages & sending in chunks we can send directly. Thus increasing the buffer size 👍

Reply:
fixed